packages/db-objects-file/lib/objects/objectsInMemServerRedis.js

```diff
@@ -728,27 +704,17 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
         const {id, namespace, name, isMeta} = this._normalizeId(pattern);
 
         if (namespace === this.namespaceObj) {
-            this.getKeys(id, (err, result) => {
-                if (err || !result) {
-                    return handler && handler.sendError(responseId, new Error(`ERROR getKeys: ${err}`));
-                }
-                for (let i = 0; i < result.length; i++) {
-                    result[i] = this.namespaceObj + result[i];
-                }
-                // if scan, we send the cursor as first argument
-                handler && handler.sendArray(responseId, isScan ? ['0', result] : result);
-            });
+            let result = this._getKeys(id);
+            result = result.map(val => this.namespaceObj + val);
```

Comment:
Does `namespaceObj` end with a dot?

Reply:
yes, why?

Reply:
otherwise you would have to add `+ '.' +` ;)